packages/expo-notifications/android/src/test/java/expo/modules/notifications/service/delegates/ExpoHandlingDelegateTest.kt

```diff
@@ -101,7 +101,7 @@ class ExpoHandlingDelegateTest {
         NotificationContent.Builder().setTitle(title).setText(text).setBody(body).build(),
         object : NotificationTrigger {
           override fun describeContents(): Int = -1
-          override fun writeToParcel(dest: Parcel?, flags: Int) {}
+          override fun writeToParcel(dest: Parcel, flags: Int) {}
```

Comment:
Ran into a compiler error here. Parcel is not defined as nullable in the interface